src/ToastController.js

```diff
@@ -50,6 +61,8 @@ export class ToastController extends Component<Props, State> {
       <Transition appear mountOnEnter unmountOnExit timeout={time} {...props}>
         {transitionState => (
           <Toast
+            onMouseOver={() => this.onMouseOver()}
```

Comment:
```suggestion
            onMouseOver={this.onMouseOver}
```

Reply:
ahh yes that makes sense - I changed it just now 👍

Reply:
Any reason you went with `onMouseOver` rather than `onMouseEnter`?

Reply:
Not really, but as I think about it I will change it just to be on the safe side regarding bubbling events